components/common/SocialsList.vue

```diff
@@ -1,98 +1,113 @@
 <template>
-  <div>
-    <strong class="h6 my-2 ms-3 title">Contribute</strong>
-    <nav class="social">
-      <ul>
-        <li v-if="page.editLink !== false">
-          <a :href="editLink" target="_blank"><Github /> Edit this page</a>
-        </li>
-        <li>
-          <a href="https://kestra.io/slack" target="_blank"
-            ><Slack /> Join us on Slack</a
-          >
-        </li>
-        <li>
-          <a href="https://github.com/kestra-io/kestra" target="_blank"
-            ><Github /> GitHub</a
-          >
-        </li>
-        <li>
-          <a href="https://twitter.com/kestra_io" target="_blank"
-            ><twitter /> Twitter</a
-          >
-        </li>
-        <li>
-          <a href="https://www.linkedin.com/company/kestra" target="_blank"
-            ><linkedin /> LinkedIn</a
-          >
-        </li>
-      </ul>
-    </nav>
-  </div>
+    <div>
+        <strong class="h6 my-2 ms-3 title">Contribute</strong>
+        <nav class="social">
+            <ul>
+                <li v-if="page.editLink !== false">
+                    <a :href="editLink" target="_blank">
+                        <Github/>
+                        Edit this page</a>
+                </li>
+                <li>
+                    <a href="https://kestra.io/slack" target="_blank"
+                    >
+                        <Slack/>
+                        Join us on Slack</a
+                    >
+                </li>
+                <li>
+                    <a href="https://github.com/kestra-io/kestra" target="_blank"
+                    >
+                        <Github/>
+                        GitHub</a
+                    >
```

Comment:
```suggestion
                    <a href="https://github.com/kestra-io/kestra" target="_blank">
                        <Github/>GitHub
                    </a>
```

Please fix that indent for everything